moveit_ros/moveit_servo/CMakeLists.txt

```diff
@@ -243,22 +246,20 @@ if(BUILD_TESTING)
 
   # Flaky, disabling until it is fixed
   # pose_tracking
-  # ament_add_gtest_executable(test_servo_pose_tracking
-  #       test/pose_tracking_test.cpp
-  # )
-  # ament_target_dependencies(test_servo_pose_tracking ${THIS_PACKAGE_INCLUDE_DEPENDS})
-  # target_link_libraries(test_servo_pose_tracking ${POSE_TRACKING})
-  # add_ros_test(test/launch/test_servo_pose_tracking.test.py ARGS "test_binary_dir:=${CMAKE_CURRENT_BINARY_DIR}")
-
-  # # low_latency
-  # add_rostest_gtest(servo_low_latency_test
-  # test/servo_low_latency_test.test
-  # test/servo_low_latency_test.cpp
-  # )
-  # target_link_libraries(servo_low_latency_test
-  # ${SERVO_LIB_NAME}
-  # ${catkin_LIBRARIES}
+  ament_add_gtest_executable(test_servo_pose_tracking
+        test/pose_tracking_test.cpp
+  )
+  ament_target_dependencies(test_servo_pose_tracking ${THIS_PACKAGE_INCLUDE_DEPENDS})
+  target_link_libraries(test_servo_pose_tracking ${POSE_TRACKING})
+  add_ros_test(test/launch/test_servo_pose_tracking.test.py ARGS "test_binary_dir:=${CMAKE_CURRENT_BINARY_DIR}")
```

Comment:
So this disables all the currently enabled servo tests and enables the pose tracking test?

Reply:
Yeah this seems to work that's why I left it open